webapp/src/lib/encoding.test.ts

```diff
@@ -0,0 +1,72 @@
+import { describe, expect, it } from "vitest";
+import {
+	bytes32ToAddress,
+	getChecksummedAddress,
+	getShortAddress,
+	compactSignatureToFullSignature,
+} from "./encoding";
+import { ethers } from "ethers";
+import type { HarbourSignature } from "./types";
+
+describe("encoding", () => {
+	const lower = "0xdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef";
+	const checksum = ethers.getAddress(lower);
```

Comment:
super nit: I would paste the string here as a constant (so not use the function call to avoid dependencies on that)